nodestream/databases/neo4j/query_executor.py

```diff
@@ -81,7 +82,16 @@ async def execute(self, query: Query, log_result: bool = False):
                 "uri": self.driver._pool.address.host,
             },
         )
-        await self.driver.verify_connectivity()
+
+        try:
+            await self.driver.verify_connectivity()
```

Comment:
The explanation in the description of the ticket could go in here as a long term comment

Reply:
Also I wonder what is the purpose of `verify_connectivity` call given that the next line is going to `execute_query`. Per the docs, `verify_connectivity` does the same thing as other methods that require connectivity:
https://neo4j.com/docs/api/python-driver/current/async_api.html#neo4j.AsyncDriver.verify_connectivity

Reply:
I'm guessing it is to establish connection so that the `execute_query` call below is faster...

Reply:
Yeah it also provides this race condition to get back to the driver before the session times out too causing all other parts of the program to have to yield the processor within 60 seconds. In the event that the database is overwhelmed/unavailable I believe this would be more useful wrapped around a backoff/retry module.